src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst

```diff
@@ -0,0 +1,252 @@
+.. _AdvancedExampleThermoPoroElasticWellbore:
+
+
+####################################################
+ThermoPoroElastic Wellbore Problem
+####################################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example uses the thermal option of the ``SinglePhasePoromechanics`` solver to handle an open wellbore problem subjected to a uniform temperature change on its inner surface. Isotropic linear thermoporoelastic behavior is considered the rock formation around the wellbore. Plane strain and axissymetric conditions are assumed.
+
+.. _problemSketchThermoPoroElasticWellboreFig:
+.. figure:: sketch.png
+   :align: center
+   :width: 500
+   :figclass: align-center
+
+   Sketch of a thermoporoelastic wellbore 
+
+Analytical solutions to this problem were first derived by `(Wang and Papamichos 1994) <https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/94WR01774>`__ using a one-way coupling simplification. They are also reformulated for the full coupling assumption in the book of `(Cheng 2016) <https://link.springer.com/book/10.1007/978-3-319-25202-5>`__. These solutions will be considered to validate GEOSX results.
```

Comment:
```suggestion
Analytical solutions to this problem were first derived by `(Wang and Papamichos 1994) <https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/94WR01774>`__ using a one-way coupling simplification. They were then reformulated for the fully coupled assumption in the book of `(Cheng 2016) <https://link.springer.com/book/10.1007/978-3-319-25202-5>`__. These solutions will be considered to validate GEOS results.
```

Reply:
The official name of the code will change back to GEOS soon. Please update the rst file and legend field in python accordingly.

Reply:
This naming change will need be done everywhere, so I propose to make the change together with the whole code to keep it consistent.